validation-test/stdlib/XCTest.swift

```diff
@@ -21,8 +21,24 @@ var XCTestTestSuite = TestSuite("XCTest")
 //       instantiate an NSInvocation with the given selector.
 
 
-func execute(_ run: () -> ()) {
+func execute(observers: [XCTestObservation] = [], _ run: () -> Void) {
```

Comment:
I think this function was originally added to get around the duplicate definitions of `XCTest.run()`. Seeing as how its role is expanding here, I wouldn't mind a rename at some point (in a future pull request maybe).